docs/api-reference/loaders-storage-targets/lake-loader/maintenance/index.md

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
I wonder if this should be inside API reference (as in this PR) or in the user-level documentation instead. @mscwilson ?

Reply:
do we have self-serve for this loader? would they get prompted as part of the set up flow to set up the maintenance jobs?